person.rb

```diff
@@ -1,7 +1,7 @@
 require './decorator'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals, :parent_permission
   attr_accessor :name, :age
```

Comment:
- [ ] Same here as well. I would recommend adding both Getters and Setters for `@rentals`

Reply:
resolved